src/graph/optimizer/OptGroup.cpp

```diff
@@ -41,9 +41,81 @@ OptGroup::OptGroup(OptContext *ctx) noexcept : ctx_(ctx) {
   DCHECK(ctx != nullptr);
 }
 
+Status OptGroup::validateSubPlan(const OptGroupNode *gn,
```

Comment:
I can't find the call to this function. Did I miss something?

Reply:
Good catch! Actually I don't use it in this PR except to debug some crash. I want to use it when refactoring go planner, now i need to consider loop/select plan nodes, it's a little tedious!

Reply:
ACK.